tutorials/lbs/first/first_example.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun, February 23
+
+Copyright (c) 2025 quocdang1998
+"""
+
+import os
+import sys
+import math
+
+sys.path.append(os.path.abspath("../../../"))
+
+from pyopensn.mesh import OrthogonalMeshGenerator, KBAGraphPartitioner
+from pyopensn.mat import AddMaterial
+from pyopensn.xs import LoadFromOpenSn
+from pyopensn.source import VolumetricSource
+from pyopensn.aquad import ProductQuadrature
+from pyopensn.solver import DiscreteOrdinatesSolver, SteadyStateSolver
+from pyopensn.fieldfunc import FieldFunctionGridBased
```

Comment:
This is comment for everybody. Should we teach users to use the submodule name, i.e. what is being done in lua now?

Example
```
mg = meshgen.OrthogonalMeshGenerator(...)
```

Reply:
i am fine with doing the import as in line 15, so line 35 looks good as is to me.